src/uu/split/src/split.rs

```diff
@@ -22,111 +19,112 @@ use std::fs::File;
 use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Read, Write};
 use std::path::Path;
 
-static NAME: &str = "split";
-static VERSION: &str = env!("CARGO_PKG_VERSION");
-// TODO: pack OPTION_… into local module?
-static OPTION_SUFFIX_LENGTH: &str = "suffix-length";
-static DEFAULT_SUFFIX_LENGTH: usize = 2;
-static OPTION_BYTES: &str = "bytes";
-static OPTION_LINE_BYTES: &str = "line-bytes";
-static OPTION_NUMERIC_SUFFIXES: &str = "numeric-suffixes";
-static OPTION_ADDITIONAL_SUFFIX: &str = "additional-suffix";
-static OPTION_FILTER: &str = "filter";
-static OPTION_LINES: &str = "lines";
-static OPTION_VERBOSE: &str = "verbose";
-
-static ARG_INPUT: &str = "input";
-static ARG_PREFIX: &str = "prefix";
+const NAME: &str = "split";
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+mod options {
```

Comment:
Well, I like this approach but I prefer consistency with the rest of the binaries ... :/

I have been trying to make them closer to each others in term of styles to facilitate contributions but this isn't the case here.

Reply:
I changed this because I wanted to do `match` on a couple of strings... will try another way around it, then

Reply:
hmm, you marked it as resolved but I still see the "mod options" ?
thanks

Reply:
to be clear, I was talking about this kind of declaration:
https://github.com/uutils/coreutils/blob/master/src/uu/mktemp/src/mktemp.rs#L30-L35

but I think yours is much better.

Would you be interested in updating the rest of the coreutils to your mod options { } style?

Reply:
oh, I misunderstood you... I thought you meant the `const` bit

I'll change it back to `static OPTION_`

Also, I'll do a separate pull request for `mod options { }`

Reply:
Excellent, thanks!